tests/scripts/init-vault.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 
-vault server -dev -dev-root-token-id="${VAULT_TOKEN}" &
+vault server -dev \
+    -dev-listen-address="${VAULT_ADDR#http://}" \  # unlike VAULT_ADDR this option does not want the protocol prefix
+    -dev-root-token-id="${VAULT_TOKEN}" \
+    &
 
 until vault status
 do
```

Comment:
yeah, I feel like I didn't setup the vault server properly, as I think it's stuck in this do-until loop